tests/platform_helper/test_command_copilot.py

```diff
@@ -80,6 +82,217 @@
 ADDON_CONFIG_FILENAME = "addons.yml"
 
 
+class TestTerraformEnabledMakeAddonCommand:
+    @pytest.mark.parametrize(
+        "kms_key_exists, kms_key_arn",
+        (
+            (True, "arn-for-kms-alias"),
+            (False, "kms-key-not-found"),
+        ),
+    )
+    @freeze_time("2023-08-22 16:00:00")
+    @patch(
+        "dbt_platform_helper.utils.versioning.running_as_installed_package",
+        new=Mock(return_value=False),
+    )
+    @patch("dbt_platform_helper.jinja2_tags.version", new=Mock(return_value="v0.1-TEST"))
+    @patch(
+        "dbt_platform_helper.commands.copilot.get_log_destination_arn",
+        new=Mock(
+            return_value='{"prod": "arn:cwl_log_destination_prod", "dev": "arn:dev_cwl_log_destination"}'
+        ),
+    )
+    @patch(
+        "dbt_platform_helper.utils.application.get_profile_name_from_account_id",
+        new=Mock(return_value="foo"),
+    )
+    @patch("dbt_platform_helper.utils.application.get_aws_session_or_abort")
+    @patch("dbt_platform_helper.commands.copilot.get_aws_session_or_abort")
+    @mock_sts
+    @mock_s3
+    @mock_iam
+    @mock_kms
+    def test_s3_kms_arn_is_rendered_in_template(
+        self, mock_get_session, mock_get_session2, fakefs, kms_key_exists, kms_key_arn
+    ):
+        client = mock_aws_client(mock_get_session)
+        mock_aws_client(mock_get_session2, client)
```

Comment:
What is `mock_get_session2` about/for? It does not appear to be used.

Reply:
it's a bit messy, but it has to mock the `get_aws_session_or_abort` in both places to work.  It is being used.